lib/dmv_facility.rb

```diff
@@ -0,0 +1,58 @@
+DATA_SOURCES = {
+  co: { name:}
+  ny: { }
+  mo: { }
+}
```

Comment:
I think a constant variable like this could be useful, but since this isn't being used anywhere else in your code, it's best to remove it. Also, if you kept it, I'd expect it to be defined inside of the class (probably right below the initialize method)